frontend/next.config.js

```diff
@@ -53,4 +53,7 @@ module.exports = {
 
     return config;
   },
+  env: {
+    NEAR_EXPLORER_DATA_SOURCE: true,
```

Comment:
I fail to understand the naming and the need for this configuration. Please, consider either renaming it or adding a detailed comment about it.

Reply:
https://github.com/near/near-explorer/pull/556
I remember this is for unit test to pass since only this way can separate browser side and client side